common/models/access-token.js

```diff
@@ -172,7 +172,12 @@ module.exports = function(AccessToken) {
     cookies = cookies.concat(['access_token', 'authorization']);
 
     for (length = params.length; i < length; i++) {
-      id = req.param(params[i]);
+      var param = params[i];
+      // replacement for deprecated req.param()
+      id = req.params && req.params[param] !== undefined ? req.params[param] :
+        req.body && req.body[param] !== undefined ? req.body[param] :
+        req.query && req.query[param] !== undefined ? req.query[param] :
+        undefined;
 
```

Comment:
I'd rather not. The utility function already exists, it's called `req.param()`, right? Ideally, we should revisit this code at the time of 3.x release and keep only the sources that make sense. For example, I am not sure if it makes sense to pass the access token in the URL path (`req.params`) . Or perhaps we should force the users to be more specific about where to get the token from (body, query, path param).